src/frameworks/drupal7/solr.md

```diff
@@ -2,29 +2,48 @@
 
 ## Requirements
 
-You will need to add the [Search
-API](https://www.drupal.org/project/search_api) and [Search API
-Solr](https://www.drupal.org/project/search_api_solr) modules to your
-project.
+You will need to add the [Search API](https://www.drupal.org/project/search_api) and [Search API
+Solr](https://www.drupal.org/project/search_api_solr) modules to your project. The [Search API Override](https://www.drupal.org/project/search_api_override) module is strongly recommended in order to allow the Solr configuration to be populated from `settings.php`.
 
 If you are using a make file, you can add those lines to your
 `project.make`:
 
 ```ini
-projects[search_api][version] = 1.13
-projects[search_api_solr][version] = 1.x-dev
+projects[entity][version] = 1.8
```

Comment:
There's a YAML syntax for Drush make? That's news to me.  Even if there, the rest of our Drush make documentation uses the ini format so let's be consistent.  Switching it would be a separate task.